src/components/Shared/AnnotationMenu/AnnotationMenuComponent.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import {IconName, MenuItem} from "@blueprintjs/core";
+import {CARTA} from "carta-protobuf";
+
+import {CustomIcon, CustomIconName} from "icons/CustomIcons";
+import {RegionStore} from "stores/Frame";
+
+interface AnnotationMenuComponentProps {
+    handleRegionTypeClicked: (type: CARTA.RegionType) => void;
+}
+
+export const AnnotationMenuComponent: React.FC<AnnotationMenuComponentProps> = ({handleRegionTypeClicked}) => {
```

Comment:
`React.FC` could be problematic when upgrading to React 18. Change it to:
`export const AnnotationMenuComponent = ({handleRegionTypeClicked}: AnnotationMenuComponentProps) => {`.